scripts/arango/js/setup.js

```diff
@@ -0,0 +1,35 @@
+
+
+print(">> Running Setup in Arangosh");
+
+const users = require('@arangodb/users');
+
+users.save(process.env.ARANGO_DB_USER, process.env.ARANGO_DB_USER_PASSWORD);
+
+db._createDatabase(process.env.ARANGO_DB_NAME);
+users.grantDatabase(process.env.ARANGO_DB_USER, process.env.ARANGO_DB_NAME, 'rw');
+
+db._useDatabase(process.env.ARANGO_DB_NAME);
+
+// TODO load this from separate file and link with app's list of db references
+require("internal").load(process.env.ARANGO_DOCKER_VOLUME_SCRIPTS_DIR + "/data/testData.js");
```

Comment:
Nice! Can we fail early with a nice Error if this file isn't found \ fails to load?